vi_cleaner/numberical_vi.py

```diff
@@ -59,7 +59,7 @@
 )
 
 _multiply_number_re = "(" + _normal_number_re + r")(x|\sx\s)(" + _normal_number_re + ")"
-_range_number_re = re.compile(_number_re + r"(-|\s\-\s)" + _end_number_re + vietnamese_without_num_re, re.IGNORECASE)
+_range_number_re = re.compile(r"(từ|tới|còn|đến)" + _number_re + r"(-|\s\-\s)" + _end_number_re + vietnamese_without_num_re, re.IGNORECASE)
```

Comment:
How about `khoảng` anh, should we put into a separate list file anh

Reply:
yeah i agree, it is getting longer now

Reply:
done